src/main/java/step5/domain/RacingGame.java

```diff
@@ -0,0 +1,35 @@
+package step5.domain;
+
+import step5.view.ResultView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingGame {
+    // Car 객체 생성 같은 것들은 일급 컬렉션에게 위임 (꿀팁 감사합니다 ㅎㅎ)
+    private final CarList carList;
+
+    public RacingGame(CarList carList) {
+        this.carList = carList;
+    }
+
+    public void start(int tryCount) {
+        List<Car> cars = this.carList.getCarList();
+
+        for (int i = 0; i < tryCount; ++i) {
+            move(cars);
+        }
+    }
+
+    // 음.. 이것도 Car 클래스 내부에 있어야 더 적합할 거 같은 느낌도 있는데 어떻게 생각하시는지 궁금합니다 !!
+    private void move(List<Car> cars) {
+        for (int i = 0; i < cars.size(); ++i) {
+            Car car = cars.get(i);
+            car.move(new RandomMovingStrategy());
+            ResultView.print(car);
+        }
+
+        System.out.println();
+    }
```

Comment:
> 음.. 이것도 Car 클래스 내부에 있어야 더 적합할 거 같은 느낌도 있는데 어떻게 생각하시는지 궁금합니다 !!

개인적으론 다음과 같이 Cars에서 수행하는게 더 어울릴 것 같은데 어떻게 생각하시나요 :) ?
`cars.move(new RandomMovingStrategy());` 라는 코드를 읽어보면
`자동차들을 움직여라` 와 같이 객체에게 명령(요청)하는 것 처럼 읽히지 않나요? ㅎㅎ
이처럼 객체지향의 세계에서는 오직 한 가지 의사소통 수단만이 존재하는데 이를 `메시지`라고 하며
상태를 외부에 노출시키지 않고 수신받은 메시지에 대해 수행함으로써 캡슐화를 지키는 과정중 하나라고 생각되어지는데
정균님께서는 어떻게 생각하시나요~?

```suggestion
    public void start(int tryCount) {
        while (tryCount-- > 0) {
            cars.move(new RandomMovingStrategy());
        }
    }
```

Reply:
오!! 저도 move 랑 print 메소드랑 위와 같이 붙어 있어서 불편하다 생각했는데 말씀해주신 거 들으니 감탄이 나옵니다 ㅠ,ㅠ  저도 나중에는 리뷰어님 처럼 사고할 수 있기를 바라며,, 열심히 리팩터링 해보겠습니다 감사합니다~~